src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```diff
@@ -43,11 +43,11 @@
     <ItemGroup>
       <_Packages Include=".\bin\$(Configuration)\*.nupkg" />
     </ItemGroup>
-    <Copy SourceFiles="@(_Packages)" DestinationFolder="../local-packages" />
+    <Copy SourceFiles="@(_Packages)" DestinationFolder="..\local-packages" />
```

Comment:
If this was a problem wouldn't it be completely broken all the time? Path issues are generally not intermittent things.